Description: Do not use namespaced oslo.concurrency
 Upstram code is using the old version of oslo.concurrency, which breaks hard.
Author: Thomas Goirand <zigo@debian.org>
Bug-Debian: https://bugs.debian.org/792034
Forwarded: not-needed
Last-Update: 2015-07-15

--- designate-2015.1.0.orig/designate/backend/agent_backend/impl_bind9.py
+++ designate-2015.1.0/designate/backend/agent_backend/impl_bind9.py
@@ -17,8 +17,8 @@ import os
 
 import dns
 import dns.resolver
-from oslo.config import cfg
-from oslo.concurrency import lockutils
+from oslo_config import cfg
+from oslo_concurrency import lockutils
 from oslo_log import log as logging
 
 from designate.backend.agent_backend import base
